packages/mobile/src/screens/user-list-screen/UserList.tsx

```diff
@@ -51,16 +52,16 @@ export const UserList = (props: UserListProps) => {
   const { tag, userSelector, setUserList } = props
   const isFocused = useIsFocused()
   const styles = useStyles()
-  const cachedUsers = useRef([])
+  const cachedUsers = useRef<User[]>([])
   const dispatchWeb = useDispatchWeb()
   const [isRefreshing, setIsRefreshing] = useState(true)
-  const { hasMore, userIds, loading } = useSelectorWeb(userSelector, isEqual)
+  const { hasMore, userIds, loading } = useSelectorWeb(userSelector)
```

Comment:
What's the reasoning for removing isEqual?

Reply:
This was a rebase conflict. Changed back. ty